tasks/tests/test_create_task.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+from tasks.models import Priority, Status, Task
+from users.models import User, Team, Role
+from django.contrib.auth.models import User as DjangoUser
+
+
+@pytest.mark.django_db
+class TestCreateTask:
+    """
+    Add test team to DB
+    """
+    @pytest.fixture
+    def team(self):
+        team = Team(name="TestTeam",
+                    description="This is a test team")
+        team.save()
+        return team
+
+    """
+    Add another test team to DB
+    """
+    @pytest.fixture
+    def other_team(self):
+        team = Team(name="TestOtherTeam",
+                    description="This is a test team")
+        team.save()
+        return team
+
+    """
+    Add manager user to the DB
+    """
+    @pytest.fixture
+    def manager(self, team):
+        django_user = DjangoUser.objects.create_user(username="TestManager",
+                                                     email="example@gmail.com",
+                                                     password='xsdDS23',
+                                                     first_name="Test",
+                                                     last_name="Test")
+        manager = User.objects.create(user=django_user,
+                                      role=Role.MANAGER,
+                                      team_id=team)
+        manager.save()
+        return manager
+
+    """
+    Add an employee to the team in "team" fixture
+    """
+    @pytest.fixture
+    def employee(self, team):
+        django_user = DjangoUser.objects.create_user(username="TestEmployee",
+                                                     email="example@gmail.com",
+                                                     password='xsdDS23',
+                                                     first_name="Test",
+                                                     last_name="Test")
+        employee = User.objects.create(user=django_user,
+                                       role=Role.EMPLOYEE,
+                                       team_id=team)
+        employee.save()
+        return employee
+
+    """
+    Add an employee to the team in "other_team" fixture
+    """
+    @pytest.fixture
+    def employee_other_team(self, other_team):
+        django_user = DjangoUser.objects.create_user(username="TestEmployee",
+                                                     email="example@gmail.com",
+                                                     password='xsdDS23',
+                                                     first_name="Test",
+                                                     last_name="Test")
+        employee = User.objects.create(user=django_user,
+                                       role=Role.EMPLOYEE,
+                                       team_id=other_team)
+        employee.save()
+        return employee
+
+    """
+    Test add new task
+    """
+    def test_add_to_db(self, manager, employee):
```

Comment:
maybe something more precise
```suggestion
    def test_add_task_to_db(self, manager, employee):
```